sdk/ai/azure-ai-generative/azure/ai/generative/synthetic/qa.py

```diff
@@ -204,20 +211,43 @@ def export_to_file(self, output_path: str, qa_type: QAType, results: Union[List,
         if not isinstance(results[0], List):
             results = [results]
         
-        for qs_and_as in results:
-            chat_history = []
-            for question, answer in qs_and_as: 
-                if qa_type == QAType.CONVERSATION:
-                    # Chat History columns:
-                    data_dict["chat_history"].append(json.dumps(chat_history))
-                    data_dict["chat_input"].append(question)
-                    chat_history.append({"inputs": {"chat_input": question}, "outputs": {"chat_output": answer}})
-                else:
-                    # QnA columns:
-                    data_dict["question"].append(question)   
+        if output_format == OutputStructure.PROMPTFLOW:
+            
+            if qa_type == QAType.CONVERSATION and not ("chat_history_key" in field_mapping and "question_key" in field_mapping and "answer_key" in field_mapping):
+                raise Exception("Field mapping for Promptflow output with Conversation must contain following keys: chat_history_key, question_key, answer_key")
+            elif not ("question_key" in field_mapping):
```

Comment:
how about answer key in this part?

Reply:
I think we should return chat_history column in all QATypes and it could be empty list when QAType != conversation. This is because PF may have a chathistory input field but user may want to evaluate it on just long/short answer for certain metrics. PF evaluation UI does not allow skipping chat history if it is not part of input file.

Reply:
I set it up to add chat history in all cases, but since it is not something that has to be considered with non-conversational questions, I didn't set it as a required key